Template/Layout.tex

```diff
@@ -7,6 +7,7 @@
 
 \documentclass[11pt,a4paper]{article}
 
+\usepackage[italian]{babel}
```

Comment:
Conviene mettere, in ordine:
1. fontenc
2. inputenc
3. babel

Mentre l'ordine dei primi due mi sembra già meno scontato, direi che babel [sicuramente va per terzo](https://guitex.org/home/en/forum/5-tex-e-latex/70373-inputenc-e-fontenc-quale-dei-due-prima).

L'ordine corrisponde alle iniziali di Federazione Italiana Basket. :D (fontenc, inputenc, babel)